MaeYoungPhan/MdBayGameReact
# set up initial components and views. Created managers director and au…

…th modules. Edited default app.js and css to BayGame and edited index.js. Login/Logout/Register working.

# Description of Proposed Changes
Login/Logout/Register features added

Edited default app and index files
Added starter components and files for nav, auth, views and managers

---


# Steps to Test

[] Fetch branch and switch to it, npm start
[] Fetch corresponding branch for server and start debugger
[] Logout if logged in
[] Click register and send request to register. You should be logged in after clicking register.
[] Logout and try to login as user you just registered
[] Logout and try to login as existing user in database
[] Verify database auth_user table in server contains newly registered user
[] Proofread files changed
